Sources/StreamChat/APIClient/Endpoints/Payloads/ChannelListPayload.swift

```diff
@@ -265,9 +265,6 @@ public class ChannelConfig: Codable {
     public let createdAt: Date
     /// A channel updated date.
     public let updatedAt: Date
-    
-    /// Determines if users are able to flag messages. Enabled by default.
-    public var flagsEnabled: Bool { commands.map(\.name).contains("flag") }
```

Comment:
Can we put it back but with a deprecation warning saying it always returns `false` and will be removed soon? Just to avoid a breaking change